src/native/qgsnative.h

```diff
@@ -135,6 +135,11 @@ class NATIVE_EXPORT QgsNative : public QObject
      */
     virtual void setApplicationBadgeCount( int count );
 
+    /**
+     * This is used to force the use of the dark theme
```

Comment:
I'd just suggest updating the this to

"Returns true if the operating system is set to utilise a "dark" theme."